packages/blockchain-info-components/src/Colors/Default.ts

```diff
@@ -50,7 +50,7 @@ const Default = {
   orange700: '#D96716',
   orange800: '#B24400',
   orange900: '#8C2F00',
-  red000: '#FAECEB',
+  red000: '#FFECEB',
```

Comment:
https://github.com/blockchain/components/blob/master/src/Tools/palette.js#L41

Did you mean to change this?

Reply:
https://www.figma.com/file/Nks0uPvdzq4hOTPkvuebhK/Inputs?node-id=0%3A1 <--- looks like design changed red000?